spec/models/book_spec.rb

```diff
@@ -18,4 +18,8 @@
     it { should validate_presence_of(:published_at) }
     it { should validate_presence_of(:cover_url) }
   end
+
+  describe "assosiasions" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.